backend/benefit/users/api/v1/views.py

```diff
@@ -31,3 +40,33 @@ def _get_current_user(self, request):
         if settings.NEXT_PUBLIC_MOCK_FLAG and isinstance(request.user, AnonymousUser):
             set_mock_user_name(request.user)
         return request.user
+
+
+class UserUuidGDPRAPIView(GDPRAPIView):
+    """
+    GDPR-API view that is used from Helsinki-profiili to query what data this app has
+    on person.
+    """
+
+    permission_classes = [BFGDPRScopesPermission]
+    authentication_classes = []
+
+    def get_object(self) -> AbstractBaseUser:
```

Comment:
So, this works now that if you are authenticated as someone and know someone else Helsinki-profile UUID, you can query that person's data. Of course, this is not production ready, but it would be nice to see at this point that the auth even works in a testing environment at all.